library-management-system/pom.xml

```diff
@@ -2,22 +2,28 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Please revert this file from the PR and make sure it is ignored by git using the .gitignore file

Reply:
Can you please also revert all files under target and make sure they are ignored by git using .gitignore file?